test/routes/mail/index.spec.js

```diff
@@ -0,0 +1,70 @@
+import nock from 'nock';
+import assert from 'power-assert';
+import request from 'supertest';
+import app from 'app.js';
+
+const SLACK_POST = process.env.KINDLIZED_SLACK;
+
+describe('/routes/mail', ()=> {
```

Comment:
__[jshint]__
*  'arrow function syntax (=>)' is only available in ES6 (use esnext option).